src/api/app/controllers/webui/project_controller.rb

```diff
@@ -206,10 +206,10 @@ def show
     @comment = Comment.new
     render :show, status: params[:nextstatus] if params[:nextstatus]
 
-    # TODO: Remove the `return unless` and the flash once this should be available to all beta users
-    return unless User.current && User.current.in_beta? && (User.current.is_admin? || User.current.is_staff?)
+    # TODO: Remove the `return unless` and the flash once this should be available to all beta users on all environments
+    return unless User.current && User.current.in_beta? && (Rails.env.development? || Rails.env.test?)
```

Comment:
why do we need this for test?

Reply:
Because we should test it :thinking:

Reply:
We will write some feature specs once we start to have a bit more content in the Bootstrap version of project pages.

Reply:
until now there are no tests and we don't test the views

Reply:
But we test workflows and for that you need the view :smile:

Reply:
I would add it when it is necessary. No test need this at the moment. But I have no strong opinion